cmd/registry/patch/api.go

```diff
@@ -33,7 +33,6 @@ type ApiData struct {
 	RecommendedDeployment string           `yaml:"recommendedDeployment,omitempty"`
 	ApiVersions           []*ApiVersion    `yaml:"versions,omitempty"`
 	ApiDeployments        []*ApiDeployment `yaml:"deployments,omitempty"`
-	Artifacts             []*Artifact      `yaml:"artifacts,omitempty"`
```

Comment:
This one is unfortunate - I have WIP that will put it right back.

Reply:
Just reverted the commit to avoid churn. I incorrectly assumed this would end up in our pile of TODOs for a while still.

Reply:
No worries, I should have left that alone - it's easy enough to add back.